File: freecodecamp/daily-code-challenge/2026-03-07.py
def get_element_size(window_size, element_vw, element_vh):
    # Extract window width and height
    w, h = map(int, window_size.split(" x "))
    
    # Extract percentage values
    vw_percent = float(element_vw.replace("vw", ""))
    vh_percent = float(element_vh.replace("vh", ""))
    
    # Calculate pixel sizes
    element_width = int(w * vw_percent / 100)
    element_height = int(h * vh_percent / 100)
    
    return f"{element_width} x {element_height}"

File: freecodecamp/daily-code-challenge/2026-03-07.js
function getElementSize(windowSize, elementVw, elementVh) {

  let [w, h] = windowSize.split(" x ").map(Number);

  // Extract numeric percentage values
  let vwPercent = parseFloat(elementVw);
  let vhPercent = parseFloat(elementVh);

  // Calculate pixel sizes
  let elementWidth = (w * vwPercent) / 100;
  let elementHeight = (h * vhPercent) / 100;

  return `${elementWidth} x ${elementHeight}`;
}
